### PR TITLE
Set Static LOGGERS

### PR DESCRIPTION
**Pull request by iancdiana21**

### CODE REVIEW

File: src/main/java/ro/db/vendor/controller/VendorController.java [hunk 1]
```diff
@@ -35,14 +35,14 @@
...
-  public Logger LOG = Logger.getLogger(VendorController.class);
-  private VendorService vendorService;
-  private SkillService skillService;
-  private VendorSkillService vendorSkillService;
-  private VendorManagerService vendorManagerService;
-  private HolidayService holidayService;
-  private OvertimeService overtimeService;
-  private VendorDaysOffService vendorDaysOffService;
+  public static final Logger LOG = Logger.getLogger(VendorController.class);
+  private final VendorService vendorService;
```
Comment (alexandrubirlan) [inline src/main/java/ro/db/vendor/controller/VendorController.java]: why do we make this final?

**Comment by alexandrubirlan** [comment]
I agree with @vsilviu . Lombok can spare you of a lot of boilerplate code. Check it out.
However, I believe that it's good for you to have a better understanding at what should happen behind the scenes in this stage of your development.

**Comment by alexandrubirlan** [comment]
"final" is fine for the LOG.
The question is why do you set final on the injected services. This forces you to inject only through the constructor which might look nice however I think it might lead to problem when creating unit tests and mocking as well as tight coupling.
I'm in doubt as well and this is not the best place to start debating.
Check this out: https://kinbiko.com/java/dependency-injection-patterns/

**Comment by iancdiana21** [comment]
Yes, you're right! I didn't want to use final for the services, it was a mistake. I will take a look over Lombok. Thank you for your advice! @alexandrubirlan @vsilviu :-)

**Comment by iancdiana21** [comment]
[[Refactor] Logger - using Lombok](https://trello.com/c/pbIOgq2R/61-refactor-logger-using-lombok)

**Pull request merged**